A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml

```diff
@@ -29,7 +28,6 @@
 				<Checkpoint ReferenceId="APU_Bleed_On"/>
 				<Checkpoint ReferenceId="A320_PREP_CHECK_OVRHEAD"/>
 				<Checkpoint ReferenceId="A320_PREP_XBLEED"/>
-				<Checkpoint ReferenceId="A320_PREP_WEATHER_RADAR"/>

```

Comment:
I would keep this one? Removing the RPM3 & Center Tank Pump is indeed fine (I'd personally even remove them from the Library.xml; it's in version history anyway)

Reply:
Not in the SOP at that point, hence removed.

Reply:
Where did you check this? Can you maybe add a URL in the comments to a reference? I could only find various quite different versions of company-specific checklists.

Reply:
https://cdn.discordapp.com/attachments/740273811617874043/752276022254370847/A32NX_Project_Standard_Operating_Manual_1.pdf Not 100% consistent with this right now but it should for the most part say the same stuff.